.eleventy.js

```diff
@@ -23,10 +24,14 @@ module.exports = function (config) {
 
   // Specific scripts to guides
   config.addPassthroughCopy("./assets/_common/js/*");
-  //
+
   config.addPassthroughCopy({'./assets/_common/_img/favicons/favicon.ico': './favicon.ico' });
   config.addPassthroughCopy({'./assets/_common/_img/favicons': './img/favicons' });
 
+  // Download folders
+  // TODO: Is this the right way to configure it?
+  config.addPassthroughCopy("./assets/**/dist/*");
```

Comment:
This appears to be the first guide that provides files to download. I set up this line to let you link to, for instance, "/assets/brand/dist/file-to-download.zip", and have it work. I have a hunch that this is not quite right, and that there might be an established pattern for handling downloads in 11ty, but this works and it's straightforward.